src/libs/ajax.js

```diff
@@ -300,15 +288,6 @@ export const DAR = {
     return await res.json();
   },
 
-  getDarFields: async (id, fields) => {
-    let url = `${await Config.getApiUrl()}/api/dar/find/${id}`;
-    if (fields !== null) {
-      url = url + `?fields=${fields}`;
-    }
-    const res = await fetchOk(url, Config.authOpts());
-    return await res.json();
-  },
-
```

Comment:
This means that we can remove the endpoint here: https://github.com/DataBiosphere/consent/blob/develop/src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java#L94

Reply:
this is already being deleted in DUOS-1315 which is now unblocked bc of this approval